data/src/main/java/io/gnosis/data/repositories/UnstoppableDomainsRepository.kt

```diff
@@ -0,0 +1,38 @@
+package io.gnosis.data.repositories
+
+import com.unstoppabledomains.config.network.model.Network
+import com.unstoppabledomains.exceptions.ns.NamingServiceException
+import com.unstoppabledomains.resolution.DomainResolution
+import com.unstoppabledomains.resolution.Resolution
+import com.unstoppabledomains.resolution.naming.service.NamingServiceType
+import io.gnosis.data.BuildConfig
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asEthereumAddress
+import java.util.concurrent.Callable
+import java.util.concurrent.FutureTask
+
+class BackgroundTask : Callable<String> {
```

Comment:
this should not be needed, because repository's resolve function is marked as suspend. So please delete this BackgroundTask